packages/combobox/src/Menu.styles.ts

```diff
@@ -0,0 +1,80 @@
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { fontFamilies, typeScales, Mode } from '@leafygreen-ui/tokens';
+import { ComboboxSize } from './Combobox.types';
+import { menuItemHeight, menuItemPadding } from './constants';
+
+/**
+ * Menu styles
+ */
+
+export const popoverStyle = (width = 384) => css`
+  border-radius: 4px;
+  width: ${width}px;
+
+  & > * {
```

Comment:
Yeah, this is pretty gross. I'll fix that

Reply:
Best we can do here is `& > div` since we need to select the container div that `Popover` injects. Can't get around it with `overflow:hidden` since that'll get rid of the shadow

Reply:
Shouldn't we apply the style through the className prop on Popover then?